st2common/st2common/policies/concurrency.py

```diff
@@ -35,6 +35,6 @@ def _get_status_for_policy_action(self, action):
         if action == 'delay':
             status = action_constants.LIVEACTION_STATUS_DELAYED
         elif action == 'cancel':
-            status = action_constants.LIVEACTION_STATUS_CANCELED
+            status = action_constants.LIVEACTION_STATUS_CANCELING
```

Comment:
The st2action worker https://github.com/StackStorm/st2/blob/master/st2actions/st2actions/worker.py will process the execution and let the appropriate runner do the cancel. For most non-mistral actions, there's no specific cancel procedures and it just inherits and runs from the base runner cancel method which is empty. https://github.com/StackStorm/st2/blob/master/st2common/st2common/runners/base.py#L106 After the cancel is run, the status of execution will be changed to CANCELED. https://github.com/StackStorm/st2/blob/master/st2actions/st2actions/container/base.py#L154